lit/v5.2.0.lit

```diff
@@ -0,0 +1,111 @@
+\use-plugin{concourse-docs}
+
+\release-version{5.2.0}
+\release-date{2019-5-XX}
```

Comment:
I think these can actually be omitted now that we just use these files to generate Markdown text and put it in the release (which has its own release date and has the version as the title already).

Reply:
somehow when I removed the release-version and the release-date I ended up with an extra `4` before the paragraph. Had to leave the release version to make it work.